SPARCore/GADArticle.m

```diff
@@ -2,6 +2,48 @@
 
 @implementation GADArticle
 
++ (void) fetchArticlesWithPublication: (NSString *)publicationId
+                                        completionHandler:(void(^_Nonnull)(NSArray<GADRemoteModel *>
+                                                                           *_Nullable models,
+                                                            NSError *_Nullable error))completion {
+    //Should we create NSDictionary parameters within each method and just request the publicationId
+    //from the front end?
+
+}
+
+- (void) populateArticleWithId: (NSString *)articleId
+                                        completionHandler:(void(^_Nonnull)(GADRemoteModel *
+                                                                           *_Nullable models,
+                                                            NSError *_Nullable error))completion {
+    //Similarly, should we create NSDictionary parameters within each method and just request the articleId
+    //from the front end?
+    
+}
+
++ (NSArray <GADArticle *> *) articlesFromJSON: (NSData *)json {
```

Comment:
I'm not really sure how we'd do it (just spent a few minutes searching) so maybe we'll just need to impress upon @AlexanderOtavka how important it is that the JSON we get always is in array form (otherwise this will probably crash).